src/compilation_error.rs

```diff
@@ -103,7 +103,7 @@ impl Display for CompilationError<'_> {
       DuplicateRecipe { recipe, first } => {
         writeln!(
           f,
-          "Recipe `{}` first defined on line {} is redefined on line {}",
+          "TDV: Recipe `{}` first defined on line {} is redefined on line {}",
```

Comment:
TDV that is my initials.  Its a place holder to remind me to come back here and do something.  Its like a TODO but at this point I don't know what should be done.